packages/block-editor/src/components/index.js

```diff
@@ -48,6 +48,7 @@ export { default as __experimentalDuotoneControl } from './duotone-control';
 export { default as __experimentalFontAppearanceControl } from './font-appearance-control';
 export { default as __experimentalFontFamilyControl } from './font-family';
 export { default as __experimentalLetterSpacingControl } from './letter-spacing-control';
+export { default as TextAlignmentControl } from './text-alignment-control';
```

Comment:
I didn't add the `__experimental` prefix, does it matter? Or maybe we should expose it as a private component initially.

Reply:
`__experimental` doesn't mean a lot now that we have to support backwards compatibility for any `__experimental` components that make their way into WP Core.

You could consider making it a private API if you'd like, but I don't think the component you're adding is very controversial or likely to change so I think it's fine to be public from day one.